### ISSUE TITLE
Tracheobronchomegaly

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11038]
**Preferred term label:**
Tracheobronchomegaly

**Synonyms**

**Definition (free text, please give PubMed ID)**
dilatation of the intrathoracic trachea and of the major bronchi

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mounier-Kuhn syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal  - this is bundled, we would use the combination of 
Tracheomegaly HP:0010778
Bronchomegaly HP:0010777
